go/prometheusexporter/webserver/webserver.go

```diff
@@ -38,19 +41,36 @@ func serveAliveness(w http.ResponseWriter, r *http.Request) {
 }
 
 func serveAppMetrics(w http.ResponseWriter, r *http.Request) {
-	vars := mux.Vars(r)
-	app := vars["application"]
-	env := vars["environment"]
+	app, env := getAppEnv(r)
+	h := collectormanager.GetRequestHandler(app + "-" + env)
+	serve(w, r, h.ServeHTTP)
+}
+
+func serveExceptionsMetrics(w http.ResponseWriter, r *http.Request) {
+	app, env := getAppEnv(r)
+	h := collectormanager.GetCollector(app + "-" + env)
+	serve(w, r, h.ServeExceptionsMetrics)
+}
+
+func serve(w http.ResponseWriter, r *http.Request, serveFn serveMetrics) {
+	app, env := getAppEnv(r)
 	h := collectormanager.GetRequestHandler(app + "-" + env)
```

Comment:
This call is repeated in the serve method.

Reply:
I refactored it a little bit to reduce duplication.

Reply:
accepted